packages/compass/src/app/components/home.tsx

```diff
@@ -213,7 +214,13 @@ function Home({
           <CompassSettingsPlugin></CompassSettingsPlugin>
           <CompassFindInPagePlugin></CompassFindInPagePlugin>
           <AtlasAuthPlugin></AtlasAuthPlugin>
-          <CompassGenerativeAIPlugin></CompassGenerativeAIPlugin>
+          <CompassGenerativeAIPlugin
+            projectId={
```

Comment:
@Anemy this was throwing an error because the plugin takes this argument in compass-web so it can be passed to the AIOptInModal Component within the AtlasAIPlugin. So is this an okay way to handle this or should i be passing an empty string always because it will never get used by compass electron since the AIOptInModal isn't shown there (and the only use of this parameter is to create the url for the project settings page for the optin modal)

Reply:
just fixed it to be the latter!